test/lib/next-modes/base.ts

```diff
@@ -446,9 +448,9 @@ export class NextInstance {
       }
 
       if (!process.env.NEXT_TEST_SKIP_CLEANUP) {
-        await fs.rm(this.testDir, { recursive: true, force: true })
+        rmSync(this.testDir, { recursive: true, force: true })
```

Comment:
```suggestion
        // Faster than `await fs.rm`. Benchmark before change.
        rmSync(this.testDir, { recursive: true, force: true })
```

because I've seen countless refactors where people convert sync fs ops to async ones thinking you can infinitely parallelize fs ops.